include/atari_screen_charmap.h

```diff
@@ -0,0 +1,304 @@
+/*                                                                           */
+/*                                                                           */
+/* (C) 2016 Christian Krueger                                                */
+/*                                                                           */
+/*                                                                           */
+/* This software is provided 'as-is', without any expressed or implied       */
+/* warranty.  In no event will the authors be held liable for any damages    */
+/* arising from the use of this software.                                    */
+/*                                                                           */
+/* Permission is granted to anyone to use this software for any purpose,     */
+/* including commercial applications, and to alter it and redistribute it    */
+/* freely, subject to the following restrictions:                            */
+/*                                                                           */
+/* 1. The origin of this software must not be misrepresented; you must not   */
+/*    claim that you wrote the original software. If you use this software   */
+/*    in a product, an acknowledgment in the product documentation would be  */
+/*    appreciated but is not required.                                       */
+/* 2. Altered source versions must be plainly marked as such, and must not   */
+/*    be misrepresented as being the original software.                      */
+/* 3. This notice may not be removed or altered from any source              */
+/*    distribution.                                                          */
+/*                                                                           */
+/*****************************************************************************/
+
+/* No include guard here! Multiple use in one file may be intentional. */
+
+#pragma charmap (0x00, 0x40)
+#pragma charmap (0x01, 0x41)
+#pragma charmap (0x02, 0x42)
+#pragma charmap (0x03, 0x43)
+#pragma charmap (0x04, 0x44)
+#pragma charmap (0x05, 0x45)
+#pragma charmap (0x06, 0x46)
+#pragma charmap (0x07, 0xFD)
+#pragma charmap (0x08, 0x48)
+#pragma charmap (0x09, 0x7F)
+#pragma charmap (0x0A, 0xDB)
+#pragma charmap (0x0B, 0x4B)
+#pragma charmap (0x0C, 0x7D)
+#pragma charmap (0x0D, 0x4D)
+#pragma charmap (0x0E, 0x4E)
+#pragma charmap (0x0F, 0x4F)
+
+#pragma charmap (0x10, 0x50)
+#pragma charmap (0x11, 0x51)
+#pragma charmap (0x12, 0x52)
+#pragma charmap (0x13, 0x53)
+#pragma charmap (0x14, 0x54)
+#pragma charmap (0x15, 0x55)
+#pragma charmap (0x16, 0x56)
+#pragma charmap (0x17, 0x57)
+#pragma charmap (0x18, 0x58)
+#pragma charmap (0x19, 0x59)
+#pragma charmap (0x1A, 0x5A)
+#pragma charmap (0x1B, 0x5B)
+#pragma charmap (0x1C, 0x5C)
+#pragma charmap (0x1D, 0x5D)
+#pragma charmap (0x1E, 0x5E)
+#pragma charmap (0x1F, 0x5F)
+
+#pragma charmap (0x20, 0x00) 
```

Comment:
Both that `#pragma` and the first one in the list should be surrounded by `#pragma warn (remap-zero)` lines (we don't want cc65 to warn us constantly about something that we are doing deliberately):
```c
#pragma warn (remap-zero, push, off)
#pragma charmap (0x20, 0x00)
#pragma warn (remap-zero, pop)
```
(You don't need to change "atari_atascii_charmap.h".)